doc/user_guide/color_usage.rst

```diff
@@ -0,0 +1,1525 @@
+        w = chart_dict['width']
+        h_l = 50
+        h_s = 20
+        cr_l = chart_dict['cornerRadius_large']
+        cr_s = chart_dict['cornerRadius_small']
+        no = chart_dict['no_colors']
+
+        # define color scale and stroke
+        color_scale = chart_color_scale(scheme_name, dict_schemes, continuous, custom, no, method)
+
+        # define how hex-codes can be derived, from datasource or using color scale
+        if single:
+            expr_hex = "datum.hex" if custom and not continuous else "scale('color', datum.i)"
+        elif top:
+            expr_hex = "datum.hex" if custom and not continuous else "scale('concat_0_color', datum.i)"
+        elif bottom:
+            expr_hex = "datum.hex" if custom and not continuous else "scale('concat_1_color', datum.i)"
+
+        # define styling for y-axis
+        my_y_axis = alt.Axis(
+            grid=False,
+            ticks=False,
+            labels=False,
+            domain=False,
+            titleAngle=0,
+            titleAlign="right",
+            titleBaseline="middle",
+        )    
+
+        # determine color pallettes for different color deficiencies
+        normal_vision = chart_normal_vision(data, h_l, w, expr_hex, my_y_axis, cr_l, no)
+        green_blindness = chart_green_blindness(data, h_s, w, expr_hex, my_y_axis, cr_s, no)
+        red_blindness = chart_red_blindness(data, h_s, w, expr_hex, my_y_axis, cr_s, no)
+        monochrome = chart_grayscale(data, h_s, w, expr_hex, my_y_axis, cr_s, no)
+
+        # determine which color pallettes to return
+        if cvd and grayscale:
+            chart_concat = (normal_vision & green_blindness & red_blindness & monochrome & color_scale)
+        elif cvd and not grayscale:
+            chart_concat = (normal_vision & green_blindness & red_blindness & color_scale)
+        else:
+            chart_concat = (normal_vision & color_scale)
+
+        # set configuration on concatenated chart object
+        if single:
+            chart_concat = chart_concat.configure_concat(spacing=1).configure_view(stroke=None)    
+        return chart_concat
+
+
+The basics of color selection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Color selection is an essential aspect of creating clear and informative
+visualizations of scientific data. To make the most effective choices,
+it's important to understand the basics of color perception and theory.
+
+Color perception and theory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We will explain the basics using two colors, ``blue`` and ``orange``: 
+ 
+.. altair-plot::
+    :remove-code:
+
+    percep_schemes = {
+        "two_color": ['blue','orange']
+    }    
+    plot_scheme("two_color", percep_schemes, cvd=False, continuous=False, method='hcl').configure_scale(rectBandPaddingInner=0.1)
+
+Now, lets say we want to define a gradient between ``blue`` and ``orange``.
+That means we have to go gradually from ``blue`` to ``orange``. 
+The interpolation between colors can follow several methods.
```

Comment:
```suggestion
Now, lets say we want to define a gradient between ``blue`` and ``orange``.
That means we have to gradually transition from ``blue`` to ``orange``,
which is also called an "interpolation" between the two colors.
```

Then after this I would have the paragraph that starts with "So what is the correct interpolation...", then show the color space plot, then show the ppaths and the corresponding sequential color schemes created by each path.